src/ucs/sys/topo/base/topo.c

```diff
@@ -29,6 +29,7 @@
 #define UCS_TOPO_SYSFS_DEVICES_ROOT  "/sys/devices"
 #define UCS_TOPO_DEVICE_NAME_UNKNOWN "<unknown>"
 #define UCS_TOPO_DEVICE_NAME_INVALID "<invalid>"
+#define UCS_TOPO_MAX_DEPTH           8
```

Comment:
move to `ucs_topo_iface_lowest_device_path` (as it is only used there).

Reply:
Done